tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,12 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    
+    assert len(items) == 0
+    assert item_a not in items
+    assert item_c not in items
+    assert item_b not in items
```

Comment:
Great asserts! These three aren't necessary since the first one tells us there's nothing in the list. We don't need to check each individual item.